### DISCUSSION TITLE
Confused on documentation in MetroWindow and SaveWindowPosition

### DISCUSSION THREAD

**Discussion by msavazzi** [MahApps/MahApps.Metro#4040] [Q&A] [answered]
Team in the MetroWindow documentation I read:

> Properties
> One property not detailed is the SaveWindowPosition="True|False" (default False) option. Setting this property to True will mean on next launch, it will automatically be positioned and sized to what it was on exit. This is designed to improve UX and speed development as its one of those "plumbing" UI things that is done regularly.
> 
> Be careful though - if a monitor is detached during application exit and restart, or if certain circumstances arise, your application may launch off screen. Be sure to provide a 'reset' option or handle that in code.

But what does this mean?

**Be careful though - if a monitor is detached during application exit and restart, or if certain circumstances arise, your application may launch off screen. Be sure to provide a 'reset' option or handle that in code.**

I'm confused on how to do it. I would expect that MetroWindow if is opened outside of the screen is reset to the standard options. Am I wrong?
I love the functionality by the way :)

**Comment by timunie** [comment, marked as answer]
Hi @msavazzi 

I never saw this in my Apps nor any user reported this to me and I heavily use this feature. Nevertheless there *may* be a case which no one considered and then there *might* be an issue. 

This may also happen to other Apps: https://www.howtogeek.com/howto/windows/bring-misplaced-off-screen-windows-back-to-your-desktop-keyboard-trick/ 

Happy coding
Tim
